exec/cnex/global.c

```diff
@@ -78,6 +78,7 @@ static FILE *check_file(TExecutor *exec, void *pf)
     return f;
 }
 #define DWORD int32_t
+#define GetLastError() 0x02
```

Comment:
Surely this can't possibly be the right thing to do...

Reply:
It's sort of a hack to make the compiler happy until go back and implement the file module.